File: backend/src/main/java/com/lifeos/telemetry/sync/dto/SyncResponse.java
package com.lifeos.telemetry.sync.dto;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SyncResponse {

    private boolean success;

    private int syncedSessions;

    private int syncedLogs;

    private String message;
}

File: android/app/src/main/java/com/karthik/lifeos/tracker/sync/SyncResponse.kt
package com.karthik.lifeos.tracker.sync

/**
 * Response from POST /api/sync.
 * Matches the backend's SyncResponse DTO structure.
 */
data class SyncResponse(
    val success: Boolean,
    val syncedSessions: Int,
    val syncedLogs: Int,
    val message: String?
)
